.changeset/wild-dodos-rush.md

```diff
@@ -0,0 +1,4 @@
+---
+'@openfn/language-dhis2': patch
+---
+Removed support for DHIS2 v42
```

Comment:
This changeset needs updating. Is this the old changeset from the last fixes?

Anyway this needs to be a major update

Reply:
If this changelog doesn't apply it needs to be removed, because it'll create an inaccurate release note and confuse users